docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md

```diff
@@ -102,27 +102,27 @@ You can debug programs that run on Azure virtual machines by using Server Explor
 
 ### To debug an Azure virtual machine
 
-1. In Server Explorer, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
+1. In **Server Explorer**, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
 
-2. Open the context menu and select **Enable Debugging**. When asked if you're sure if you want to enable debugging on the virtual machine, select **Yes**.
+2. Go to the context menu and select **Enable Debugging**. When asked if you're sure if you want to enable debugging on the virtual machine, select **Yes**.
 
     Azure installs the remote debugging extension on the virtual machine to enable debugging.
 
     ![Virtual machine enable debugging command](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746720.png)
 
     ![Azure activity log](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746721.png)
 
-3. After the remote debugging extension finishes installing, open the virtual machine's context menu and select **Attach Debugger...**
+3. After the remote debugging extension finishes installing, go to the virtual machine's context menu and select **Attach Debugger...**
```

Comment:
I would keep the original.

Reply:
The guideline about using "go to" to open menus is here: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui.
Does it apply in this context?